### PR TITLE
[Doc] Add explanations for setting geo-replication at topic level

### PR DESCRIPTION
**Pull request by Anonymitaet**
Add docs for https://github.com/apache/pulsar/pull/12136

This PR adds doc for 2.8.2 and master. Will add docs for 2.9.1 once the doc set is generated.

Preview looks good:

image

image

### CODE REVIEW

File: site2/docs/administration-geo.md [hunk 1]
```diff
@@ -113,11 +115,21 @@ $ bin/pulsar-admin namespaces set-clusters my-tenant/my-namespace \
...
+```shell
+$ bin/pulsar-admin topics set-replication-clusters --clusters us-west,us-east,us-cent my-tenant/my-namespace/my-topic
+```
 
-Once you create a geo-replication namespace, any topics that producers or consumers create within that namespace is replicated across clusters. Typically, each application uses the `serviceUrl` for the local cluster.
+> **Tip**
+> 
+> - You can change the replication clusters for a namespace at any time, without disruption to ongoing traffic. Replication channels are immediately set up or stopped in all clusters as soon as the configuration changes.
+>
+> - Once you create a geo-replication namespace, any topics that producers or consumers create within that namespace is replicated across clusters. Typically, each application uses the `serviceUrl` for the local cluster.
```
Comment (Huanli-Meng) [inline site2/docs/administration-geo.md:130]:
```suggestion
> - Once you create a geo-replication namespace, any topics that producers or consumers create within that namespace are replicated across clusters. Typically, each application uses the `serviceUrl` for the local cluster.
```

**Review by Huanli-Meng — APPROVED** [review]
Overall LGTM. only one minor comment.

File: site2/website/versioned_docs/version-2.8.2/administration-geo.md [hunk 2]
```diff
@@ -144,12 +156,22 @@ producer.newMessage()
 
 #### Topic stats
 
-Topic-specific statistics for geo-replication topics are available via the [`pulsar-admin`](reference-pulsar-admin.md) tool and {@inject: rest:REST:/} API:
+You can check topic-specific statistics for geo-replication topics using one of the following methods.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--pulsar-admin-->
+
+Use the [`pulsar-admin persistent stats`](https://pulsar.apache.org/tools/pulsar-admin/) command.
```
Comment (Technoboy-) [inline site2/website/versioned_docs/version-2.8.2/administration-geo.md:164]: Here, we may need to change the command to `pulsar-admin topics stats`

File: site2/website/versioned_docs/version-2.8.2/administration-geo.md [hunk 3]
```diff
@@ -144,12 +156,22 @@ producer.newMessage()
...
 
 ```shell
 $ bin/pulsar-admin persistent stats persistent://my-tenant/my-namespace/my-topic
```
Comment (Technoboy-) [inline site2/website/versioned_docs/version-2.8.2/administration-geo.md:167]: Also here to `pulsar-admin topics stats`

**Review by Technoboy- — APPROVED** [review]

**Comment by codelipenghui** [comment]
@Anonymitaet @Huanli-Meng @Technoboy- Enable or disable geo-replication on the topic level will release at 2.8.3 and 2.9.1, so we don't need to add doc for 2.8.1, 2.8.2 (since @315157973 already started the 2.8.2 release.)

**Review by codelipenghui — APPROVED** [review]

**Pull request merged**